android-o/AndroidCipher/AndroidCipher/Constants.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace AndroidCipher
+{
+    public class Constants
+    {
+        public static readonly int AesKeySize = 128;
+        public static readonly int GcmTagLength = 16;
+        public static readonly int GcmNonceLength = 12;
+
+        public static readonly string Charset = "UTF-8";
+        public static readonly string Algorithm = "AES";
+        public static readonly string Transformation = "AES/GCM/NoPadding";
```

Comment:
Do you think it makes sense to add a spinner to the app that would allow us to demonstrate multiple encryption algorithms? Looking at the 8.1 API docs it looks like there may be more options we can expose here?

Reply:
@pjcollins The main idea is to test that I can retrieve GCMParameters from encryptor to decryptor.